lib/OctokitAction.ts

```diff
@@ -68,6 +88,27 @@ export abstract class OctokitAction extends Action {
     }
   }
 
+  private async jiraTransitionRequest(method: "GET" | "POST", issueId: string, body?: any): Promise<any> {
```

Comment:
To get rid of these Jira prefixes, we should probably create a `jira` object here of a new `JiraClient` class. And all `jira` prefixed methods would live there (without the prefix). Just to isolate that logic, as there will be quite a lot.